utils/clients/settings/client_settings_factory.py

```diff
@@ -34,9 +34,9 @@ def create(self, settings_type, username, password, host, port, **kwargs):
                                                 reduction_complete, reduction_error
         :return: A ClientSettings object
         """
-        if settings_type.lower() not in ['database', 'icat', 'queue']:
-            raise ValueError("Factories creation settings type must be one of: 'database', "
-                             "'icat', 'queue'")
```

Comment:
in raise value can we change this to be 
```
valid_types = ['database', 'icat', 'queue', 'sftp']
if settings_type.lower() not in valid_types:
	raise ValueError(f"Factories creation settings type must be one of: {','.join(valid_types)}")
```

Reply:
> in raise value can we change this to be
> 
> ```
> valid_types = ['database', 'icat', 'queue', 'sftp']
> if settings_type.lower() not in valid_types:
> 	raise ValueError(f"Factories creation settings type must be one of: {','.join(valid_types)}")
> ```

I've added this change In ef9252c, but have put `valid_types` into Class scope so that it can be accessed by the test method (rather than the test method making it's own `valid_types` list)